generated/nidigital/nidigital/session.py

```diff
@@ -2415,6 +2415,28 @@ def fetch_capture_waveform(self, site_list, waveform_name, samples_to_read, time
 
         return waveforms
 
+    @ivi_synchronized
+    def get_site_pass_fail(self, site_list):
```

Comment:
This method should use rep cap.
Please merge and update.

Reply:
This will happen in #1290. Whichever PR goes in last will have to merge.
FYI: @texasaggie97